.changes/v3.0.0/710-features.md

```diff
@@ -0,0 +1,15 @@
+* Added `TmOrg` and `types.TmOrg` to structure for OpenAPI management of Organizations with methods
+  `VCDClient.CreateTmOrg`, `VCDClient.GetAllTmOrgs`, `VCDClient.GetTmOrgByName`,
+  `VCDClient.GetTmOrgById`, `TmOrg.Update`, `TmOrg.Delete`, `.TmOrg.Disable` [GH-710]
+* Added `TmVdc` and `types.TmVdc` to structure for OpenAPI management of Organizations VDCs with
+  methods `VCDClient.CreateTmVdc`, `VCDClient.GetAllTmVdcs`, `VCDClient.GetTmVdcByName`,
+  `VCDClient.GetTmVdcById`, `TmVdc.Update`, `TmVdc.Delete`, `.TmVdc.Disable` [GH-710]
+* Added types `Region` and `types.Region` for managing regions with methods
+  `VCDClient.CreateRegion`, `VCDClient.GetAllRegions`, `VCDClient.GetRegionByName`,
+  `VCDClient.GetRegionById`, `Region.Update`, `Region.Delete` [GH-710]
```

Comment:
I'm a bit hesitant to merge this PR until we can write tests for create/update/delete.